Userland/Libraries/LibRegex/RegexByteCode.cpp

```diff
@@ -569,6 +569,10 @@ ALWAYS_INLINE ExecutionResult OpCode_Compare::execute(MatchInput const& input, M
             auto general_category = static_cast<Unicode::GeneralCategory>(m_bytecode->at(offset++));
             compare_general_category(input, state, general_category, current_inversion_state(), inverse_matched);
 
+        } else if (compare_type == CharacterCompareType::Script) {
+            auto script = static_cast<Unicode::Script>(m_bytecode->at(offset++));
+            compare_script(input, state, script, current_inversion_state(), inverse_matched);
+
```

Comment:
I know you're just following the existing style, but this is weird - there's no need for a random empty line before a closing curly brace. When you add a section here next time, consider removing these 😄